tests/conftest.py

```diff
@@ -1,13 +1,25 @@
+import os
 import pytest
+
 from new_arrivals_chi.app.main import app as flask_app
 
 
 @pytest.fixture(scope="module")
 def app():
     """Provides the Flask application instance configured for testing."""
+    flask_app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///:memory:"
+    flask_app.config["SECRET_KEY"] = os.getenv("SECRET_KEY")
```

Comment:
These values are being set twice in your code. I would determine how to do this once, as this could create some frustrating debug issues down the line.